frontend/packages/operator-lifecycle-manager/src/components/descriptors/spec/configure-update-strategy.tsx

```diff
@@ -13,7 +14,7 @@ export const configureUpdateStrategyModal = ({
     resourceKind: kindObj,
     resource,
     defaultValue: specValue,
-    title: `Edit ${specDescriptor.displayName}`,
+    title: i18n.t('olm~Edit {{item}}', { item: specDescriptor.displayName }),
```

Comment:
Should content in descriptor strategy update modal be translated as well? It seems most content and text are hard coded in this modal so that we should translate them too.
<img width="455" alt="Screen Shot 2021-01-08 at 2 35 22 PM" src="https://user-images.githubusercontent.com/12692381/103982739-0cc87800-51bf-11eb-8f5c-b39cc8e71759.png">